package.json

```diff
@@ -200,7 +200,8 @@
     "webpack-virtual-modules": "0.4.3",
     "worker-loader": "3.0.8",
     "yaml-lint": "1.7.0",
-    "yarn": "1.22.18"
+    "yarn": "1.22.18",
+    "eslint-plugin-local-rules": "link:./eslint-plugin-local-rules"
```

Comment:
We make the local rules a linked dev dependency. This allows eslint to search for the rules as a plugin.

Reply:
We might want to consider documenting this somewhere. I can see how someone might encounter this link in the future and have questions about why we are doing this.